### PR TITLE
Update latest release to 3.12.5

### PR DESCRIPTION
**Pull request by efritz**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -114,7 +114,7 @@ All notable changes to Sourcegraph are documented in this file.
 ### Fixed
 
 - The `/.auth/saml/metadata` endpoint has been fixed. Previously it panicked if no encryption key was set.
-- The version updating logic has been fixed for `sourcegraph/server`. Users running `sourcegraph/server:3.12.3` will need to manually modify their `docker run` command to use `sourcegraph/server:3.12.3` or higher. [#7442](https://github.com/sourcegraph/sourcegraph/issues/7442)
+- The version updating logic has been fixed for `sourcegraph/server`. Users running `sourcegraph/server:3.12.5` will need to manually modify their `docker run` command to use `sourcegraph/server:3.12.5` or higher. [#7442](https://github.com/sourcegraph/sourcegraph/issues/7442)
```
Comment (unknwon) [inline CHANGELOG.md]: I think this should be remain 3.12.3, it’s a changelog entry.

File: doc/index.md [hunk 2]
```diff
@@ -49,7 +49,7 @@ We actively maintain the two most recent monthly releases of Sourcegraph.
 Upgrades should happen across consecutive minor versions of Sourcegraph. For example, if you are
 running Sourcegraph 3.1 and want to upgrade to 3.3, you should upgrade to 3.2 and then 3.3.
 
-> The Docker server image tags follow SemVer semantics, so version 3.8 can be found at `sourcegraph/server:3.12.3`. You can see the full list of tags on our [Docker Hub page](https://hub.docker.com/r/sourcegraph/server/tags).
+> The Docker server image tags follow SemVer semantics, so version 3.8 can be found at `sourcegraph/server:3.12.5`. You can see the full list of tags on our [Docker Hub page](https://hub.docker.com/r/sourcegraph/server/tags).
```
Comment (unknwon) [inline doc/index.md]: Geoffrey mentioned this should be 3.8.2 in previous reverted PR.

Comment (efritz) [reply doc/index.md]: Seems like `yarn run release release:publish 3.12.5` is over zealous.

File: doc/index.md [hunk 3]
```diff
@@ -49,7 +49,7 @@ We actively maintain the two most recent monthly releases of Sourcegraph.
 Upgrades should happen across consecutive minor versions of Sourcegraph. For example, if you are
 running Sourcegraph 3.1 and want to upgrade to 3.3, you should upgrade to 3.2 and then 3.3.
 
-> The Docker server image tags follow SemVer semantics, so version 3.8 can be found at `sourcegraph/server:3.12.3`. You can see the full list of tags on our [Docker Hub page](https://hub.docker.com/r/sourcegraph/server/tags).
+> The Docker server image tags follow SemVer semantics, so version 3.8 can be found at `sourcegraph/server:3.8.2`. You can see the full list of tags on our [Docker Hub page](https://hub.docker.com/r/sourcegraph/server/tags).
```
Comment (uwedeportivo) [inline doc/index.md]: ha ! nice catch

File: doc/index.md (hunk 2 above)
Comment (unknwon) [reply doc/index.md]: It’s surprising that we’ve increased this number form 3.8.2 to this far 😅

**Pull request merged**